routes/truckRoutes.js

```diff
@@ -1,3 +1,60 @@
 // connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
 // use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
 // remember this is a Node module
+var express = require('express');
+var mongoose = require('mongoose');
+var Truck = require('../models/truckModel');
+
+var router = express.Router();
+
+router.route('/')
+	.get(function(request, response) {
+		Truck.find(function(error, trucks) {
+			if(error) {
+				response.status(500).send(error);
+			} else {
+				response.send(trucks);
+			}
+		});
+	});

```

Comment:
When you chain route handlers, you don't want to terminate the ones preceding others with a semi-colon.